test/active_model/xml_serialization_test.rb

```diff
@@ -248,4 +281,11 @@ def to_ary
     xml = @contact.to_xml(include: :contact)
     assert xml.include?(%(<contact type="SerializableContact">))
   end
+
+  test "computed property applies only to root" do
+    role = Role.new("manager")
+    human = Human.new("Jane", "Air", role)
+    xml = human.to_xml(methods: :full_name)
+    assert_match %r{<full-name>}, xml
```

Comment:
Will be good add "not to rise", but I didn't found minitest matcher for this quickly.